server/public/styles.css

```diff
@@ -58,30 +58,53 @@ body {
 }
 
 .location-form,
-.caption-form {
-  width: 100%;
+.caption-form,
+.date-form {
+  width: 90%;
```

Comment:
This 90% everywhere is super sketchy looking. I get the feeling you actually want to add padding to a parent element instead. A lot of your elements have width on them, but i get the feeling it's not actually needed. Please carefully examine all of your CSS, I think it's already out of control.